<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPassword</class>
 <widget class="QWidget" name="ForgetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>199</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="user_lable">
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="firstPassword_label">
        <property name="text">
         <string>登陆密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="twicePassword_label">
        <property name="text">
         <string>密码确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="user_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="firstPassword_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="twicePassword_Edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="push_Button">
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cannel_Button">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
